examples/miral-shell/shell_main.cpp

```diff
@@ -132,13 +134,23 @@ int main(int argc, char const* argv[])
       }
     };
 
+    auto const to_focus_stealing = [](bool focus_stealing_prevention)
+    {
+        if (focus_stealing_prevention)
+            return FocusStealing::prevent;
+        else
+            return FocusStealing::allow;
+    };
 
     return runner.run_with(
         {
             CursorTheme{"default:DMZ-White"},
             WaylandExtensions{},
             X11Support{},
             ConfigureDecorations{},
+            pre_init(ConfigurationOption{[&](bool is_set)
+                    { focus_stealing_prevention = to_focus_stealing(is_set); },
+                    "focus-stealing-prevention", "allow or prevent focus stealing"}),
```

Comment:
The semantics of "allow or prevent focus stealing" true are unclear, as is the absence of a default.
```suggestion
            pre_init(ConfigurationOption{[&](bool is_set)
                    { focus_stealing_prevention = to_focus_stealing(is_set); },
                    "focus-stealing-prevention", "prevent focus stealing", true}),
```

Reply:
That's on me, I thought just having a flag would be simpler, but that wouldn't allow for disabling it.

Reply:
Well, a flag could also work (it just changes from the default) I just don't like both the description and the default being unclear